sdk/search/azure-search-documents/azure/search/documents/_headers_mixin.py

```diff
@@ -4,20 +4,15 @@
 # license information.
 # --------------------------------------------------------------------------
 
-from typing import TYPE_CHECKING
+from typing import Optional, Dict
 
-if TYPE_CHECKING:
-    from typing import Optional
 
-
-class HeadersMixin(object):
+class HeadersMixin:
     @property
-    def _headers(self):
-        # type() -> dict
+    def _headers(self) -> Dict:
```

Comment:
if its known that the value of `self._credential.key` and `self._ODATA_ACCEPT` will be `str` the return can be Dict[str, str] otherwise `Dict[str, Any]`